assets/sass/components/_personalised.scss

```diff
@@ -1,60 +1,3 @@
 .hidden {
```

Comment:
I suspect this could be moved to one of the other SCSS files so you can delete this?

Reply:
Moved to _general.scss in 12cee9b6a2097c4d1f7789af34fdde580b1bab47